pages/samvera/developer_resources/collections/hyrax_2.1/nesting-faq.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "FAQ - Collection Nesting"
+keywords: Nesting, Collection, FAQ
+categories: How to Do All the Things
+permalink: collection-nesting-faq.html
+folder: samvera/how-to/collection_extensions/hyrax_2.1/nesting-faq.md
+sidebar: home_sidebar
+tags: [development_resources]
+a-z: ['FAQ - Collection Nesting', 'Collections - Nesting', 'Nesting of Collections']
+version:
+  label: 'targeted for Hyrax v2.1.0'
+  branch:
+    label: 'collections-sprint'
+    link: 'https://github.com/samvera/hyrax/tree/collections-sprint'
+---
+
+### What is collection nesting?
+
+Collections that have collections as members.
```

Comment:
This is probably too picky of me, but this just feels a bit inconsistent grammatically... Collection nesting is the ability of a collection to contain other collections as members. Nested Collections are collections that have collections as members.

Reply:
Also, it might be helpful to mention that the collection type controls whether or not a collection can be nested. I don't see any reference to collection type settings on this FAQ